projecttacoma/fqm-execution
# Integration changes

# Summary
I moved the code to execute the integration tests from under the ./src directory to the ./test/integration-test folder.
## New behavior
There should be no behavior changes and everything should function as before
## Code changes
- added tsconfig.int.json - this applies the root tsconfig.json to the test folder and extends it to ./test/integration-test
- updated the paths in multiple files that call the integration tests or are importing from src
# Testing guidance

1. run: npm install
2.  execute ./test/integration-tests/setup.sh (this will drop the files from the connectathon repo/fpg folder into the expected places)
3. Now run: npm run test:integration EXMXXX (XXX is your measure of choice from the connectathon repo 401 subfolder, I personally tested with EXM104) 

The tests should run to completion and report no issues
